Assets/_BForBoss/_Core/Scripts/Enemies/AgentNavigationBehaviour.cs

```diff
@@ -61,16 +67,27 @@ public void ResumeNavigation()
             }
         }
 
+        private bool ReachedDestination()
+        {
+            if (_agent.remainingDistance > 0)
+            {
+                return _agent.remainingDistance < _stopDistanceBeforeReachingDestination;
+            }
+
+            return false;
```

Comment:
Can rewrite this to `return _agent.remainingDistacnce > 0.0f && _agent.remaningDistance < _stopDistanceBeforeReachingDestination;`